CriteoPublisherSdk/Tests/UnitTests/CR_BidManagerTests.m

```diff
@@ -333,14 +657,7 @@ - (void)testLiveBid_GivenExpiredBid_ThenNoResponseGiven {
   CR_CdbBid *expiredBid = CR_CdbBidBuilder.new.adUnit(self.adUnit1).expired().build;
   [self givenApiHandlerRespondBid:expiredBid];
```

Comment:
Is it normal to consider expiration when bid is not put in the cache ?

Reply:
Earlier test ported from bid caching that is related to that wrong expectation that a bid could be returned expired from CDB, will remove this test (or tests?) while fixing your other comment

Reply:
Done